resources/azure-common.properties

```diff
@@ -50,4 +50,22 @@ azure.cli.version=2.53.0
 # AzurePowerShell version
 azure.powershell.version=11.5
 
+# These filters are used to implement tags for resources. Other occurrences of these resource type identifiers
+# are intentionally not filtered because doing so would unnecessarily complicate the code.
+identifier.applicationGateways=Microsoft.Network/applicationGateways
+identifier.availabilitySets=Microsoft.Compute/availabilitySets
+identifier.dnszones=Microsoft.Network/dnszones
+identifier.networkInterfaces=Microsoft.Network/networkInterfaces
+identifier.networkSecurityGroups=Microsoft.Network/networkSecurityGroups
+identifier.publicIPAddresses=Microsoft.Network/publicIPAddresses
+identifier.privateEndpoints=Microsoft.Network/privateEndpoints
+identifier.storageAccounts=Microsoft.Storage/storageAccounts
+identifier.vaults=Microsoft.KeyVault/vaults
+identifier.virtualNetworks=Microsoft.Network/virtualNetworks
+identifier.virtualMachines=Microsoft.Compute/virtualMachines
+identifier.virtualMachinesExtensions=Virtual machine extension
```

Comment:
I want to know why this one is a human readable value "Virtual machine extension" while the others are resource type identifiers.

Reply:
Hello @edburns the two resources are not recognized by the Tag control, and I want to allow the customer to provide specified tag for them. The VM deployment from portal has similar experience.

Reply:
So is this a bug in Azure, @galiacheng ?